extern/f5totec/f5totec.cpp

```diff
@@ -330,14 +322,24 @@ int main( int argc, char * argv[] ){
       btypes += ntypes;
       belem_comp_num += ntypes;
       bconn += nconn;
+      for ( int ii = 0; ii < ntypes; ii++ ){
+        belem_global_ptr[ii] = k;
+      }
+      belem_global_ptr += ntypes;
     }
 
     int num_comp = loader->getNumComponents();
 
     int *reduced_points = new int[ num_points ];
     int *reduced_conn = new int[ basic_conn_size ];
-    float *reduced_float_data = NULL;
-    reduced_float_data = new float[ num_points ];
+    int* value_location = NULL;
```

Comment:
Is there a reason for having this line as opposed to just doing `int *value_location = new int[ num_variables + num_evariables ];`?